Dockerfile.arm32v7

```diff
@@ -1,4 +1,4 @@
-FROM arm32v7/node:8-slim
+FROM arm32v7/node:15.5.1-alpine3.10
```

Comment:
>node:15.5.1-alpine3.10 [](start = 13, length = 22)

Why switching from slim to alpine? We could use this one for example - [https://hub.docker.com/layers/arm32v7/node/10-slim/images/sha256-792ec870d5fc154342c02c9a911adcab24336652cf9f935199d4ef6d8669f610?context=explore](https://hub.docker.com/layers/arm32v7/node/10-slim/images/sha256-792ec870d5fc154342c02c9a911adcab24336652cf9f935199d4ef6d8669f610?context=explore)